src/db/chainIndexers/getChainIndexer.ts

```diff
@@ -42,7 +43,7 @@ export const getBlockForIndexing = async ({
     FROM
       "chain_indexers"
     WHERE
-      "chainId"=${chainId}
```

Comment:
needed to wrap `${chainId}` in quotes to work for a SQL string comparison. Opted to use `Prisma.sql` which is the recommended way of templating in SQL string.

Reply:
Hm why can't this line just be:
```
"chainId"="${chainId.toString()}"
```

Reply:
yeah that works, and given that way know `chainId` is sanitised it should be fine, but just a standard best practice to not use raw string interpolation with SQL

Reply:
Lets simplify it. From how I'm reading Prisma docs ([source](https://www.prisma.io/docs/orm/prisma-client/using-raw-sql/raw-queries#queryraw)):
> The method is implemented as a [tagged template](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#tagged_templates), which allows you to pass a template literal where you can easily insert your [variables](https://www.prisma.io/docs/orm/prisma-client/using-raw-sql/raw-queries#using-variables). In turn, Prisma Client creates prepared statements that are safe from SQL injections:
```
const email = 'emelie@prisma.io'
const result = await prisma.$queryRaw`SELECT * FROM User WHERE email = ${email}`
```